stage/services/salesforce-api/salesforce-api_runner.js

```diff
@@ -1,3 +1,25 @@
+function hasBillingInfo(res = new Object({})) {
+  return (
+    res && res.attributes && (
+      res.attributes.billing_organization ||
+      res.attributes.billing_department ||
+      res.attributes.billing_street ||
+      res.attributes.billing_city ||
+      res.attributes.billing_state_code ||
```

Comment:
The state is not always required. Only for US, China and some other countries, see https://github.com/datacite/bracco/blob/cd6e57c731ed1699c3368802b3b9a9514a6df470/app/controllers/providers/new.js#L9